css/about.css

```diff
@@ -137,14 +134,14 @@ button {
     padding-bottom: 10px;    
 
 }
+
 .bottom {
     display: flex;
     flex-wrap: wrap;
     justify-content: flex-start;
     padding: 25px;
     padding-top: 0px;
     margin-top: 40px;
-    
 }
 
 footer {
```

Comment:
you could put all of this into the index file, this way you wont have to rewrite the nav or footer css. you could separate it by a comment so you know when each page begins and ends